tests/functional-tests/test_chart.py

```diff
@@ -10,6 +10,8 @@
 """
```

Comment:
Please run `black` on this file or `pre-commit run --all-files`

Reply:
Pre-commit check passed, I have pre-commit installed and applied and CI passed too, no?

Reply:
Saw your other PR :) , thanks. Can we merge this before that so we can run pre-commit and do a mass refactor

Reply:
This PR still needs an approval from someone with merge permissions.

Reply:
FYI [I opened a PR to add pre-commit to CI](https://github.com/astronomer/airflow-chart/pull/223). We can probably hold off on fixing pre-commit errors in this branch, then fix any follow-ups in that PR before merging, just to avoid having merge conflicts.

Reply:
Sorry, lagging here. You can merge it if you want. I was just trying to keep things easier for everybody but I don't have a preference on the order of merges.